Assets/Scripts/PlayerLevelEditor.cs

```diff
@@ -75,14 +94,12 @@ private void PlaceTile()
     private void RemoveTile()
     {
         string nameOfCurrentTile = $"{tile[currentTile].name}(Clone)";
-        Sprite deleteTile = this.transform.GetChild(0).gameObject.GetComponent<SpriteRenderer>().sprite;
         Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
         RaycastHit2D rayHit = Physics2D.Raycast(ray.origin, ray.direction, Mathf.Infinity);
```

Comment:
PlaceTile() and RemoveTile() both use a raycast, instead of creating two raycasts perhaps consider a method that they both call with parameters that can be changed